src/boosting/gbdt.h

```diff
@@ -180,6 +199,10 @@ class GBDT: public Boosting {
   data_size_t bag_data_cnt_;
   /*! \brief Number of traning data */
   data_size_t num_data_;
+  /*! \brief Number of classes */
+  int num_class_;
+  /*! \brief Current class */
+  int curr_class_;
```

Comment:
curr_class_ is used in Bagging(), UpdateScoreOutOfBag(), TrainOneIter() and UpdateScore(). It indicates current tree_learner we are using. It works just like iter_.